people/affiliations.json

```
@@ -174,5 +174,9 @@
     "Lowaiz": {
         "name": "Lyonel Martinez",
         "company": "Numberly"
+    },
+    "AhmedAmin": {
```

```suggestion
    "ahmedameenaim ": {
```
This needs to be your github handle.